src/email/service.ts

```diff
@@ -97,3 +97,42 @@ export const sendArchiveStewardNotification = async (
     ]
   );
 };
+
+export const sendInvitationNotification = async (
+  fromEmail: string,
+  toEmail: string,
+  message: string,
+  giftAmount: number,
+  token: string
+): Promise<void> => {
+  const fullNameResult = await db.sql<{ fullName: string }>(
+    "email.queries.get_full_name_by_account_email",
+    { email: fromEmail }
+  );
+  if (fullNameResult.rows[0] === undefined) {
+    throw new Error(`Account with primary email ${fromEmail} not found`);
+  }
+  const { fullName } = fullNameResult.rows[0];
+  await sendEmail(
+    "invitation-from-relationship",
```

Comment:
For some reason the email I got locally looks different than the one I got from the backend invitation.

stela:
![Screenshot 2023-10-27 at 15-52-19 Emptied database again is gifting you $10 of storage - cecilia@permanent org - The Permanent Legacy Foundation Mail](https://github.com/PermanentOrg/stela/assets/1497818/3c92bf69-7f29-4cdc-b6d8-8aa84a176661)


back-end:
![Screenshot 2023-10-27 at 15-52-35 remodel is gifting you $10 of storage - cecilia@permanent org - The Permanent Legacy Foundation Mail](https://github.com/PermanentOrg/stela/assets/1497818/d5c77c78-93aa-426e-9216-73a87ea6c6bc)


But this looks like the right template name, so I'm not sure what's going on.

Reply:
Oh duh it looks like my PR (https://bitbucket.org/permanent-org/email/pull-requests/53) broke the template so the email is only sending the first part of the text...I'll fix that.

Reply:
Fix: https://bitbucket.org/permanent-org/email/pull-requests/55